governance/program/src/state/account_governance.rs

```diff
@@ -0,0 +1,102 @@
+//! Program Governance Account
+
+use crate::{id, state::enums::GovernanceAccountType, tools::account::AccountMaxSize};
+use borsh::{BorshDeserialize, BorshSchema, BorshSerialize};
+use solana_program::{program_pack::IsInitialized, pubkey::Pubkey};
+
+/// Governance config
+#[repr(C)]
+#[derive(Clone, Debug, PartialEq, BorshDeserialize, BorshSerialize, BorshSchema)]
+pub struct GovernanceConfig {
+    /// Governance Realm
+    pub realm: Pubkey,
+
+    /// Account governed by this Governance. It can be for example Program account, Mint account or Token Account
+    pub governed_account: Pubkey,
+
+    /// Voting threshold in % required to tip the vote
+    /// It's the percentage of tokens out of the entire pool of governance tokens eligible to vote
+    pub vote_threshold: u8,
```

Comment:
nit: Since it's a percentage, we can make it clear by calling it `vote_threshold_percentage`

Reply:
Done